02. 1, 2차원 배열 탐색/03. 가위바위보/김유경/index.js

```diff
@@ -1,7 +1,22 @@
 function solution(a, b) {
-  let answer = '';
+  let answer = [];
 
-  return answer;
+  function checkVictory(a, b) {
+    if (a === b) {
+      return "D";
+    }
+    if ((a === 1 && b === 3) || (a === 2 && b === 1) || (a === 3 && b === 2)) {
+      return "A";
+    } else {
+      return "B";
+    }
```

Comment:
위에서 조건에 만족하면 얼리 리턴을 해주고 있기 때문에 else문을 안 넣어도 될 것 같습니다~
```suggestion
    }
    return "B";
```

Reply:
불필요한 else문! 제거해보겠습니다 :)